source/common/upstream/upstream_impl.h

```diff
@@ -72,6 +72,38 @@ class HealthCheckHostMonitorNullImpl : public HealthCheckHostMonitor {
   void setUnhealthy(UnhealthyType) override {}
 };
 
+/**
+ * Implementation of LoadMetricStats.
+ */
+class LoadMetricStatsImpl : public LoadMetricStats {
+public:
+  LoadMetricStatsImpl() = default;
+
+  void add(const std::string& key, double value) override {
+    absl::MutexLock lock(&mu_);
```

Comment:
suggest moving impls to .cc file.

Reply:
Done.

Reply:
Impls still in .h file?

Reply:
For some reason I misread this as "impls to impl file" originally and got confused.